src/Components/v2/Seo/Person.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react"
+import { Meta } from "react-head"
+
+export const Person = ({ data }) => {
```

Comment:
Why move this to `v2/Seo` folder? Is it used in a few spots already?

Reply:
That was where a similarly generic [Product](https://github.com/artsy/reaction/blob/8aaff476798ba25f6ba18921b551df4ee7114442/src/Components/v2/Seo/Product.tsx) existed. This feels very similar.